tasks/taxon_id/contamination/task_kraken2.wdl

```diff
@@ -141,13 +84,16 @@ task kraken2_standalone {
   output {
     String kraken2_version = read_string("VERSION")
     String kraken2_docker = docker
-    String analysis_date = read_string("DATE")
+    String kraken2_analysis_date = read_string("DATE")
     File kraken2_report = "~{samplename}.report.txt"
     File kraken2_classified_report = "~{samplename}.classifiedreads.txt.gz"
     File kraken2_unclassified_read1 = "~{samplename}.unclassified_1.fastq.gz"
     File? kraken2_unclassified_read2 = "~{samplename}.unclassified_2.fastq.gz"
     File kraken2_classified_read1 = "~{samplename}.classified_1.fastq.gz"
     Float kraken2_percent_human = read_float("PERCENT_HUMAN")
+    Float kraken2_percent_sc2 = read_float("PERCENT_SC2")
+    String kraken2_percent_target_organism = read_string("PERCENT_TARGET_ORGANISM")
+    String? kraken2_target_organism = target_organism
     File? kraken2_classified_read2 = "~{samplename}.classified_2.fastq.gz"
     String kraken2_database = kraken2_db
```

Comment:
We're currently exposing the database location, which is currently hosted in a theiagen-owned bucket and can be downloaded directly from the datatable by the user. This would be a good thing if it wasn't for egress charges. Should we take an approach followed already with other workflows (pangolin, for example) and report only the database name?

Reply:
example:
![image](https://github.com/user-attachments/assets/5576ebc0-3650-4e7d-8910-e7d9ed44af2b)

Reply:
Resolving as it was discussed during dev sync and it was agreed to keep the current behaviour